config/plugins/visualizations/common/templates/observable_entry_point.mako

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+<%inherit file="visualization_base.mako"/>
+
+## Create a container, attach data and import script file
+<%def name="get_body()">
+    <%
+        from markupsafe import escape
+
+        ## Collect incoming data
+        data_incoming = {
+            "root": h.url_for("/"),
+            "visualization_id": visualization_id,
+            "visualization_name": visualization_name,
```

Comment:
Do you need to escape the name as well ?

Reply:
Hey, thanks for looking into this. The visualization name does not need to be escaped since it's a static value specified in the visualizations.xml file. The title is escaped since its user-specified.